bot/resources/tags/listcomps.md

```diff
@@ -10,7 +10,7 @@ Using list comprehensions can make this both shorter and more readable. As a lis
 >>> [n ** 2 for n in range(5)]
 [0, 1, 4, 9, 16]
 ```
-List comprehensions also get an `if` statement:
+List comprehensions also get an `if` expression:
```

Comment:
Why this change?

Reply:
Statements must be separated by newlines or semicolons, that's why.

Reply:
Neither statement nor expression seem like appropriate descriptions since the `if` is not a standalone thing in the context of a comprehension. The [documentation](https://docs.python.org/3/reference/expressions.html#displays-for-lists-sets-and-dictionaries) calls them "`if` clauses".

Reply:
I could modify so it's "clause", not "expression" nor "statement".